api/v1/vector_store.proto

```diff
@@ -0,0 +1,223 @@
+syntax = "proto3";
+
+package llmoperator.vectore_store.v1;
+
+import "google/api/annotations.proto";
+
+option go_package = "github.com/llm-operator/file-manager/api/v1";
+
+// The API specification follows OpenAPI API specification (https://platform.openai.com/docs/api-reference/vector-stores).
+
+message ExpiresAfter {
+    // Anchor timestamp after which the expiration policy applies. Supported anchors: last_active_at.
+    string anchor = 1;
+    // The number of days after the anchor time that the vector store will expire.
+    int64 days = 2; 
+}
+
+message VectorStore{
+    string id = 1;
+    string object = 2;
+    // The Unix timestamp (in seconds) for when the vector store was created.
+    int64 created_at = 3;
+    string name = 4;
+    // The total number of bytes used by the files in the vector store.
+    int64 usage_bytes = 5;
+    int64 file_counts = 6;
```

Comment:
This should be a message? 
<img width="661" alt="image" src="https://github.com/llm-operator/file-manager/assets/8353550/714b11c8-8215-40fb-a7ee-39ccc638a48a">